SwiftLeeds/Views/Components/CommonTileButtton.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  CommonTileButtton.swift
+//  SwiftLeeds
+//
+//  Created by Alex Logan on 05/07/2022.
+//
+
+import SwiftUI
+
+struct CommonTileButtton<BackgroundType: ShapeStyle>: View {
+    let primaryText: String
```

Comment:
Is there another initialiser or should these be made private?

Reply:
The other initialisers are just conveniences, and making the above all private removes the ability to use the initialiser you get for free with a struct, so I avoid doing so.